webapp/tests/fixtures/snapcraft_forum_mock.py

```diff
@@ -23,6 +23,1206 @@ def register_uris(adapter):
     """
 
     # Register mock URIs
+    adapter.register_uri(
```

Comment:
When we have mocks we typically try to create tooling to generate them so we can easily update them when the source changes.

Reply:
that would be lovely. If you feel like writing that in a follow-up PR, it would be great.